sumo_mgmt.py

```diff
@@ -78,7 +78,7 @@ def check_alive_before_days_range(arg):
 parser.add_argument('-updateSource', metavar='', type=str, nargs=2, help='updates source with a given name to a source defined in provided JSON file')
 parser.add_argument('-listOfflineCollectors', metavar="[1-100]", type=check_alive_before_days_range, nargs=1, help='list offline collectors, with given aliveBeforeDays parameter (in range from 1 to 100)')
 parser.add_argument('-deleteOfflineCollectors', metavar="[1-100]", type=check_alive_before_days_range, nargs=1, help='delete given set of offline collectors with given aliveBeforeDays parameter (in range from 1 to 100)')
-
+parser.add_argument('-budgetId', metavar='', type=str, nargs=1, help='enter Ingest budget Id')
```

Comment:
since the other commands have the format of verbs, could we change `-budgetId` to instead be `-assignBudget` ?

Reply:
Done